.stylelintrc.yaml

```diff
@@ -25,3 +26,4 @@ rules:
   - true
   - border-in-box-model: false
     empty-line-between-groups: false
+  # TODO: Think about border property in a Box Model group
```

Comment:
Change `border` property order group to Box model. It belongs to it first of all and only after that, we can think about it as a visual property.